varint-rs/src/lib.rs

```diff
@@ -491,12 +489,14 @@ pub fn decode<R: Read, T: Default + DecoderHelper>(mut input: R) -> errors::Resu
 }
 
 /// Syncronously decode a number from a `Read`
-pub fn encode<T: EncoderHelper + Bits>(input: T) -> Vec<u8> {
+pub fn encode<T: EncoderHelper + Bits>(input: T) -> Bytes {
+    use tokio_io::io::AllowStdIo;
```

Comment:
Same here.

Reply:
I would agree with changing the `std::iter` import but I think this is very much an implementation detail of this particular function, and the global imports shouldn't have to worry about it.